src/CMakeLists.txt

```
@@ -59,6 +65,13 @@ add_executable(vita-make-fself vita-make-fself.c sha256.c)
 add_executable(vita-pack-vpk vita-pack-vpk.c)
 add_executable(vita-elf-export vita-elf-export.c yamlemitter.c)
 
+if(install_rpath)
+	set_target_properties(vita-libs-gen PROPERTIES INSTALL_RPATH ${install_rpath})
+	set_target_properties(vita-elf-create PROPERTIES INSTALL_RPATH ${install_rpath})
+	set_target_properties(vita-elf-export PROPERTIES INSTALL_RPATH ${install_rpath})
+	set_target_properties(vita-make-fself PROPERTIES INSTALL_RPATH ${install_rpath})
```

`set_target_properties` can apply an option to multiple targets in parallel.
```suggestion
	set_target_properties(vita-libs-gen vita-elf-create vita-elf-export vita-make-fself 
		PROPERTIES INSTALL_RPATH ${install_rpath})
```